packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -290,9 +300,11 @@ const DiscussionThreadInput = forwardRef((props: Props, ref: any) => {
     }
   }, [])
 
-  const isActionsContainerVisible = allowTasks || allowPolls
+  const allowAddActivity = featureFlags.meetingInception
+  const isActionsContainerVisible = allowTasks || allowPolls || allowAddActivity
```

Comment:
+1 to test the PR, we have to make `allowPolls` false, but will this be changed in a later PR or will we always need to do this locally?

We haven't worked on polls for ~2 years, so I think it should be removed from this file

Reply:
Set to be always `false`